functions/behat.sh

```diff
@@ -121,7 +121,8 @@ function drupal_ti_ensure_chrome_driver() {
 	cd $DRUPAL_TI_BIN_DIR
 
 	# @todo Make version configurable.
-	wget http://chromedriver.storage.googleapis.com/2.13/chromedriver_linux64.zip
+	local CHROMEDRIVER_VERSION=`wget http://chromedriver.storage.googleapis.com/LATEST_RELEASE -qO-`
```

Comment:
Ah, ok, I'll update the PR in a moment! I'm not super knowledgable with bash, but I thought that without 'local' the variable will leak to the global scope? In any case, that's probably not a problem so I'll remove it.